src/test/java/net/masterthought/cucumber/reducers/ReportFeatureAppendableMergerTest.java

```diff
@@ -0,0 +1,41 @@
+package net.masterthought.cucumber.reducers;
+
+import net.masterthought.cucumber.json.Feature;
+import org.junit.Test;
+
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReportFeatureAppendableMergerTest {
+
+    @Test
+    public void merge_NullSafe() {
+        assertThat(new ReportFeatureAppendableMerger().merge(null)).isNotNull().isEmpty();
+    }
+
+    @Test
+    public void merge_ReturnsOriginArray() {
+        // given
+        List<Feature> origin = singletonList(new Feature());
+
+        // when
+        List<Feature> actual = new ReportFeatureAppendableMerger().merge(origin);
+
+        // then
+        assertThat(actual).isEqualTo(origin);
```

Comment:
are you comparing array or array's content?

Reply:
then are you expecting this https://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractAssert.html#isSameAs(java.lang.Object) ?

Reply:
@damianszczepanik , nice! I didn't use it before